app/src/main/java/com/yalta/repositories/LocationRepo.kt

```diff
@@ -13,17 +11,8 @@ interface LocationRepo {
 
 class RealLocationRepo : LocationRepo, RealRepo() {
     override suspend fun sendCurrentLocation(location: Location): RepoResponse<AddedLocation> {
-        val url = "${baseUrl}/location/"
-        val client = OkHttpClient()
-        val json = common.Serialization.toJson(LocationUpdate(location.latitude, location.longitude))
-        val body = RequestBody.create(MediaType.parse("text/plain"), json)
-        val request = Request.Builder()
-            .addHeader("Cookie", SessionService.session?.token!!)
-            .url(url)
-            .post(body)
-            .build()
-
-        val response = client.newCall(request).execute()
+        val body = common.Serialization.toJson(LocationUpdate(location.latitude, location.longitude))
```

Comment:
Maybe extract `common.Serialization.toJson(...` to some beauty? 
But that's just me being picky...

Reply:
Maybe someday...